src/components/header/header.tsx

```diff
@@ -8,8 +8,9 @@ import { NavigationFlyout } from './menu-flyout';
 import { Link } from '../links/links';
 import { Swoosh } from '../icons/swoosh';
 import { LanguageSwitch } from '../language-switch/language-switch';
+import { HEADER_HEIGHT_VALUE } from '../layout/theme';
 
-export const HEADER_HEIGHT = '65px';
+export const HEADER_HEIGHT = `${HEADER_HEIGHT_VALUE}px`;
```

Comment:
I placed some core numbers into the theme. One is the header height as a number which is exposed here. I didn't want to refactor `HEADER_HEIGHT` so I just left it as is but made it dependent on the new integer from the theme.